iso15118/secc/transport/udp_server.py

```diff
@@ -60,10 +61,22 @@ async def __init__.
         # Allows address to be reused
         sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
we need to change this to 0, so that we block the case where we try to create a socket for the same interface. Also add a reference to the man page: https://www.man7.org/linux/man-pages/man7/socket.7.html

Reply:
Done.